content/kubermatic/master/tutorials_howtos/OPA_integration/Default Constraints/_index.en.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "Default Open Policy Agent (OPA) Constraints"
+date = 2021-09-08T12:07:15+02:00
+weight = 20
+
++++
+
+### Default Constraints
+
+Default Constraints allow admins to conveniently apply policies to all OPA enabled clusters.
+This would allow admins an easier way to make sure all user clusters are following some policies (for example security), instead of the current way in which Constraints need to be created for each cluster separately.
+Kubermatic operator/admin creates a Constraint in the admin panel, it gets propagated to seed clusters and user clusters with OPA-integration.
+
+More Info about the KKP OPA integration can be read from these docs and tutorials,
+[OPA Integration](https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/opa_integration/)
+
+### Managing Default Constraints
+
+Default Constraints are managed similarly to Constraint through Kubermatic CRD wrappers around the Gatekeeper Constraints, the difference being that Default Constraints are managed on the admin level by Kubermatic platform admins.
+
+On Cluster Level, Default Constraints are the same as Constraints with a `default` label to differentiate them from other Constraints.
+
+Note that they can not be edited/deleted at the Cluster level.
+
+Example of a Kubermatic Default Constraint:
+
+```yaml
+apiVersion: kubermatic.k8s.io/v1
+kind: Constraint
+metadata:
+  name: ns-must-have-gk
+  namespace: cluster-bpc9nstqvk
+  labels:
+    default: ns-must-have-gk
+spec:
+  constraintType: K8sRequiredLabels
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Namespace"]	
+  parameters:
+    rawJSON: '{"labels":["gatekeeper"]}'
+```
+
+**Filtering Clusters on Default Constraints**
+
+Admins can filter clusters where they want Default constraints applied using Cloud Provider and Label Selectors filters.
+{{% notice info %}}
+This is an EE feature. 
+{{% /notice %}}
+
+The following example will make sure that the Constraint is applied only on OPA enabled Clusters with Cloud Provider `aws` or `gcp` and Label Selector `filtered: 'true'`.
+
+```yaml
+constraintType: K8sRequiredLabels
+match:
+  kinds:
+    - kinds:
+        - Namespace
+      apiGroups:
+  labelSelector: {}
+  namespaceSelector: {}
+parameters:
+  rawJSON: '{"labels":["gatekeeper"]}'
+selector:
+  providers:
+    - aws
+    - gcp
+  labelSelector:
+    matchLabels:
+      filtered: 'true'
+```
+
+### Deleting Default Constraint
+
+Deleting Default Constraint causes all related Constraints on the user clusters to be deleted as well.
+
+Note: Cluster Admins will not be able to edit/delete Default Constraints
+
+### Disabling Constraint
+
+By setting `disabled` flag to true, Kubermatic deletes the constraint from all User clusters.
+
+Note: Constraint will still be present in the user cluster namespace on the Seed Cluster for future use.
+
+To Enable Default Constraints again, you can just remove the `disabled` flag or set it to `false`.
```

Comment:
Add a screenshot here of where to enable/disable. Its better then saying remove filtered flag. Maybe also more screenshots in this doc (like of the Admin Panel where to add the Default Constraint)